### PR TITLE
Backend with Rails for two app hello world project

### PR DESCRIPTION
**Pull request by prius2055**
### Backend with Rails for two app hello world project

_In this project, I implemented the following;_

- [ ] Created a new Rails API app.
- [ ] Set up Postgres database
- [ ] Created a table for storing messages and created 5 different greetings.
- [ ] Created an API endpoint that selects a random greeting from the table

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,190 @@
...
+
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [ruby group capstone project](#ruby-group-capstone-project)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[ruby group capstone project\] ](#-ruby-group-capstone-project-)
```
Comment (DammyShittu) [inline README.md]: - [ ] Kindly fix your project section link.

File: README.md [hunk 2]
```diff
@@ -1 +1,190 @@
...
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Prince**
+
+- GitHub: [@prius2055](https://github.com/prius2055)
+- Twitter: [@prius2055](https://www.twitter.com/prius2055)
+- LinkedIn: [@prius2055](https://www.linkedin.com/in/princenwuke)
+
+  👤 **Mahabub Alam**
```
Comment (DammyShittu) [inline README.md]: - [ ] Kindly remove this second author since this is a solo project.

**Review by DammyShittu — COMMENTED** [review]
Hi @prius2055,

While you made a great effort in this project, I cannot proceed to review your code.

## Invalid Code Review Request

### 1. Kindly fix the linter error in the frontend PR and the changes below.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md (hunk 1 above)
Comment (prius2055) [reply README.md]: Thank you. This has been fixed

File: README.md (hunk 2 above)
Comment (prius2055) [reply README.md]: Thanks a lot

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      - name: Rubocop Report
+        run: rubocop --color
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```
Comment (rotimiazeez) [inline .github/workflows/linters.yml:30]: As stated by the previous reviewer, you have an es-lint error in your frontend PR. Kindly fix it. 👍🏽
Screenshot from 2023-08-24 12-42-28

**Review by rotimiazeez — COMMENTED** [review]
# Hi @prius2055 😞, 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project that's not passing all linter checks 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

File: .github/workflows/linters.yml (hunk 3 above)
Comment (prius2055) [reply .github/workflows/linters.yml:30]: resolved

**Review by ghost — APPROVED** [review]
Hi @prius2055 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ Backend is setup correctly 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**